update-version.sh

```diff
@@ -0,0 +1,18 @@
+#/usr/bin/env bash
+
```

Comment:
Early exit, in case of error.
```suggestion
set -euo pipefail
```